src/components/ConversationsSelect.ts

```diff
@@ -0,0 +1,36 @@
+import { joinTextChildren } from './Text';
+import { ConversationsSelect as ConversationsSelectSpec } from '@slack/types';
+import { ContainerProps } from './ContainerProps';
+import { FC } from '..';
+
+export interface ConversationSelectProps extends ContainerProps<string> {
+  initialConversation?: string;
+  actionId?: string;
```

Comment:
is `actionId` actually optional?

Reply:
yes technically you can omit it and slack will assign it a unique id. i'm not sure if there any cases where we'd WANT to omit that, so we could probably change this (and the other instances like this) to require it